integrations/access/accessrequest/app.go

```diff
@@ -144,9 +153,13 @@ func (a *App) run(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-
-	if err := a.initAccessMonitoringRulesCache(ctx); err != nil {
-		return trace.Wrap(err)
+	if len(acceptedWatchKinds) == 0 {
+		return errors.New("no acceptedWatchKinds returned")
```

Comment:
```suggestion
		return trace.BadParameter("failed to initialize resource watcher: no acceptedWatchKinds returned")
```

Reply:
I would go a step further and replace the variable name with something meaningful for the user.

Something like:
```go
trace.BadParameter("failed to initialize watcher for all the required resources: %+v", watchKinds")
```
(we can probably convert the `watchKinds` to a list of strings before printing)